tests/cloner_test.go

```diff
@@ -543,7 +547,17 @@ var _ = Describe("all clone tests", func() {
 				By("Deleting verifier pod")
 				err = utils.DeleteVerifierPod(f.K8sClient, f.Namespace.Name)
 				Expect(err).ToNot(HaveOccurred())
-			})
+
+				// preallocation settings only respected for copy
```

Comment:
No I don't think we should.  At least in this PR.  Doing host clones negates a lot of efficiencies provided by modern backend storage systems like ceph. Maybe it will make sense for users that want to preallocate=true but I think it would be better to do that via a special storage class with "thick" provisioning parameters

Reply:
> special storage class with "thick" provisioning parameters

Yeah seen those before. That would make more sense